includes/installer/InitDB.php

```diff
@@ -106,7 +106,13 @@
 		$_SESSION['Username'] = "admin";
 		$_SESSION['Name'] = "admin";
 	}
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->